en/flying/package_delivery_mission.md

```diff
@@ -0,0 +1,84 @@
+# Package Delivery Mission Planning
+
+A package delivery mission allows user to plan and execute a package delivery. Note that this is only supported for vehicles that has a [delivery mechanism hardware](../peripherals/gripper.md) configured & installed.
```

Comment:
All of this only talks about grippers. In the advanced features, you talk about winches. To what level are winches supported?

Reply:
Oh so for now winch is not supported, so technically I should remove that mention in advanced feature, but it is coming in the future.

Shall I remove the mentino?

Reply:
No, we should mention that we only support grippers right now.